geometry_msgs/msg/VelocityStamped

```diff
@@ -0,0 +1,10 @@
+# This expresses the timestamped velocity of a frame 'body_frame_id' in the reference frame 'header.frame_id' expressed from arbitrary observation pose 'observation_pose'.
+# The message is often used in simplified versions where:
+# - observation pose is not defined, i.e., it is assumed that the observation pose is the origin of the 'header.frame_id';
+# - 'body_frame_id' and 'header.frame_id' are identical, i.e., the velocity is observed and defined in the local coordinates system of the body 
+#   which is the usual use-case in mobile robotics.
```

Comment:
I think you can remove this particular line

Reply:
I am with you on this, my only concern is not to confuse people that are moving from Twist as used currently in Nav2 and ros2_control.

Reply:
```suggestion
#   as common in mobile robotics.
```

Reply:
I'm a little confused. if `body_frame_id` and `header.frame_id` are identical, shouldn't the velocity be always zero? i.e, we're asking the velocity of a body relative to itself. On the other hand, we could have `body_frame_id` and `observation_pose` (via its `frame_id`) be identical and `header.frame_id`  be something different from the two. This makes sense to me since it's asking the velocity of the body relative to whatever `header.frame_id` is but expressed in the body frame.